src/cache/cache/green_cache.cpp

```diff
@@ -0,0 +1,181 @@
+// Rafael Sá Menezes - 04/2020
+
+#include "green_cache.h"
+#include <cache/algorithms/algorithms.h>
+#include <cache/algorithms/algorithms_utils.h>
+
+void green_cache::canonize_expr(expr2tc &expr)
```

Comment:
Add short description of all methods.

Reply:
The header includes the description of the method